panther_manager/include/panther_manager/moving_average.hpp

```diff
@@ -0,0 +1,38 @@
+#ifndef PANTHER_MANAGER_MOVING_AVERAGE_HPP_
+#define PANTHER_MANAGER_MOVING_AVERAGE_HPP_
+
+#include <deque>
+
+namespace panther_manager
+{
+
+template <typename T>
+class MovingAverage
+{
+public:
+  MovingAverage(const unsigned window_size = 5, const T initial_value = T(0))
+  : window_size_(window_size), sum_(initial_value)
```

Comment:
```suggestion
  : window_size_(window_size), sum_(initial_value * static_cast<T>(window_size_))
```
Otherwise, in the beginning `get_average()` will return `1 / window_size_` of the initial_value